plasmapy/particles/particle_class.py

```diff
@@ -439,37 +472,72 @@ def _validate_arguments(argument, Z, mass_numb):
         if Z is not None and not isinstance(Z, Integral):
             raise TypeError("Z is not an integer.")
 
-    def _initialize_attrs_categories(self):
-        """Create empty collections for attributes & categories."""
+    def _store_particle_identity(self):
+        """Store the particle's symbol and identifying information."""
+        argument, mass_numb, Z = self.__inputs
+        self._validate_arguments()
+        symbol = _dealias_particle_aliases(argument)
```

Comment:
```suggestion
        symbol = parsing._dealias_particle_aliases(argument)
```

I'm curious if we can improve readability here by calling the parsing functionality directly from `parsing`.  I know you and I have different views on chunking up functionality, but when it's "too much" I feel like I'm trying to find a grain of sand in a buck of sand instead of pebble in a bucket of pebbles.  By using `parsing` directly, I feel we now at least have a clump of sand.